### PR TITLE
Newsletter

### LINKED ISSUE #77: Newsletter
Implement newsletter

### PR DESCRIPTION
**Pull request by Jordan141**
Fixes #77 

- [x] Create schema
- [x] Create subscribe route
- [x] Create unsubscribe route
- [x] Add newsletter sending functionality
- [x] Connect newsletter ejs to backend
- [x] Connect unsubscribe route to backend
- [x] Send newsletter on new article creation
- [x] Test it

### CODE REVIEW

File: app/public/scripts/utils.js [hunk 1]
```diff
@@ -34,7 +35,7 @@ function myInitCode() {
   if(deleteArticleButton) deleteArticleButton.addEventListener('click', deleteHandler)
   if (hamburger) hamburger.addEventListener('click', openHamburgerMenu)
   if(uploadArticleImageButton) uploadArticleImageButton.addEventListener('click', uploadImage)
-  
+  if(unsubscribeButton) unsubscribeButton.addEventListener('click', () => clickHandler('/unsubscribe'))
```
Comment (michaelKurowski) [inline app/public/scripts/utils.js:38]: Use `<a href="..." >`

Comment (michaelKurowski) [reply app/public/scripts/utils.js:38]: `clickHandler` can be removed in general, as you should use `a href` whenever possible in for reidrections with no additional logic

File: app/routes/articles.js [hunk 2]
```diff
@@ -55,6 +60,8 @@ router.post('/', isLoggedIn, hasAuthorRole, (req, res) => {
             req.flash('error', 'Oops! Something went wrong!')
             return res.redirect('/')
         }
+
+        sendNewsletters(article).catch(err => req.log('SendNewsletters in CREATE', err))
```
Comment (michaelKurowski) [inline app/routes/articles.js:64]: Shouldn't you receive newsletter when an article is being approved, instead of created?

**Review by michaelKurowski — CHANGES_REQUESTED** [review]

File: app/mailer.js [hunk 3]
```diff
@@ -1,7 +1,10 @@
 const nodemailer = require('nodemailer')
 
 async function init() {
-    const testAccount = await nodemailer.createTestAccount()
+    const testAccount = {
+        user: 'clementina.stiedemann59@ethereal.email',
```
[the remark this answers is not in the document]
Comment (Jordan141) [reply app/mailer.js:5]: They're inconsequential, junk vars for nodemailer email, unless someone has a keen interest in our junk testdata

File: app/routes/articles.js (hunk 2 above)
Comment (Jordan141) [reply app/routes/articles.js:64]: Good catch

**Review by michaelKurowski — APPROVED** [review]

**Pull request merged**